Kudu.Core/Environment.cs

```diff
@@ -266,6 +270,18 @@ public string SiteExtensionSettingsPath
             get { return _siteExtensionSettingsPath; }
         }
 
+        public bool AlwaysOnEnabled
+        {
+            get { return _alwaysOnEnabled; }
+        }
+
+        internal bool IsAlwaysOnEnabled()
+        {
+            string value = System.Environment.GetEnvironmentVariable(WebsiteSCMAlwaysOnEnabledKey);
+
+            return string.Compare("1", value) == 0;
```

Comment:
Also I would return true if the environment variable doesn't exist this way you can integrate it now and not wait for the azure change.